### ISSUE TITLE
Allow disabling map frame

### ISSUE THREAD

**Issue by jidanni** [GenericMappingTools/gmt#7691]
Regarding the rectangular frame around the map,
there are many variables to control it,
```
$ gmt defaults|perl -nwle 'print for /MAP_FRAME\S+/g'
MAP_FRAME_AXES
MAP_FRAME_PEN
MAP_FRAME_PERCENT
MAP_FRAME_TYPE
MAP_FRAME_WIDTH
```
and [many settings](https://docs.generic-mapping-tools.org/dev/cookbook/options.html#axes-settings) too,
but alas, [no sure fire way](https://forum.generic-mapping-tools.org/t/how-to-get-rid-of-just-the-map-frame/4144)
to simply turn it off.

**Comment by joa-quim** [comment]
I think MAP_FRAME_PEN alone will do it but I’m away from computer for 2 weeks

**Comment by jidanni** [comment]
Maybe even specifying which side(s) to disable: top, bottom, left, right, could be allowed.

**Comment by joa-quim** [comment]
Disabling map frame is achieved with **-B+n** and to drop sides simply do not request them. _i.e._ **-Bl** just plots the left side.
What is not possible is drop side plots but annotate or tick them.

**Comment by PaulWessel** [comment]
It would not be too hard for me to code a solution to this.  But reality is

1. Not a very important feature to me
2. It ranks outside the top 50 in the queue of things I need to fix or implement before we can release 6.5

So take a double-doze of patience potion for now.
